core/modules/views/plugins/views_plugin_display.inc

```diff
@@ -2735,12 +2735,12 @@ class views_plugin_display extends views_plugin {
       foreach ($this->get_handlers($type) as $key => $handler) {
         if ($handler->can_expose() && $handler->is_exposed()) {
           if ($handler->is_a_group()) {
-            if ($id != $key && $identifier == $handler->options['group_info']['identifier']) {
+            if ($id != $key && isset($handler->options['group_info']['identifier']) && $identifier == $handler->options['group_info']['identifier']) {
```

Comment:
This change is not in the respective path committed to the 3.x branch of Views 7.x.

Reply:
looks like a safe addition though.